src/server_manager/web_app/digitalocean_server.ts

```diff
@@ -313,6 +314,10 @@ function startsWithCaseInsensitive(text: string, prefix: string) {
   return text.slice(0, prefix.length).toLowerCase() === prefix.toLowerCase();
 }
 
-export function GetCityId(slug: server.RegionId): string {
+export function GetCityId(slug: RegionId): string {
```

Comment:
I think this shouldn't be exported anymore. We should expose generic city ids, not DO-specific city ids.